Makefile

```diff
@@ -1,18 +1,7 @@
 all: build
 .PHONY: all
 
-build:
-	go build github.com/openshift/library-go/pkg/...
-.PHONY: build
+include build-machinery/make/golang.mk
+include build-machinery/make/targets/openshift/deps.mk
 
-test:
-	go test github.com/openshift/library-go/pkg/...
-.PHONY: test
-
-verify:
-	hack/verify-golint.sh
-.PHONY: verify
-
-update-deps:
-	hack/update-deps.sh
-.PHONY: update-deps
+GO_BUILD_PACKAGES :=./pkg/...
```

Comment:
could be, although this way it makes sure that in case the includee accidentally uses immediate assignment (`:=`) instead of conditional one (`?=`) it would still apply